app/controllers/devise/paranoid_verification_code_controller.rb

```diff
@@ -22,9 +22,14 @@ def update
   end
 
   private
-    def resource_params
+
+  def resource_params
+    if params.respond_to?(:permit)
       params.require(resource_name).permit(:paranoid_verification_code)
+    else
+      params[scope]
```

Comment:
@hoan006 I think it is not needed. in line 14, it gets `resource_params[:paranoid_verification_code]` directly.